brh.data-commons.org/manifest.json

```diff
@@ -16,7 +16,7 @@
     "hatchery": "quay.io/cdis/hatchery:feat_multiple_paymodels",
     "indexd": "quay.io/cdis/indexd:2022.03",
     "metadata": "quay.io/cdis/metadata-service:1.6.4",
-    "portal": "quay.io/cdis/data-portal:feat_multiple_paymodels",
```

Comment:
we should keep this branch

I have rebased that branch w/ master so we should be fine